shared/src/common/types/sc-instance/sc-instance-create-response-dto.type.ts

```diff
@@ -3,7 +3,8 @@ type SCInstanceCreateResponseDto = {
   instanceId: string;
   instanceType: string;
   createdAt: string;
-  publicDNS: string;
+  publicIpAddress: string;
+  state: string;
```

Comment:
```suggestion
  state: string;
```
it should be an enum